### PR TITLE
Documentation updates to include EventStreams on ICP support

### PR DESCRIPTION
**Pull request by osowski**
Updated `application-components.md` and `backing-services.md` to include specific support for all necessary parameters and deployment examples.

### CODE REVIEW

File: docs/deployments/application-components.md [hunk 1]
```diff
@@ -197,19 +195,24 @@ docker push <private-registry>/<image-namespace>/order-command-ms:latest
...
     * `--namespace <target-namespace>`
     * `--output-dir <local-template-directory>`
-
-```shell
-# Example parameters
-helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-command-ms --set image.tag=latest --set image.pullSecret= --set image.pullPolicy=Always --set eventstreams.env=ICP --set eventstreams.brokersConfigMap=kafka-brokers --set serviceAccountName=kcontainer-runtime --output-dir templ --namespace eda-refarch chart/ordercommandms/
-```
+  Example using Event Streams via ICP4I:
+   ```shell
+   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-command-ms --set image.pullSecret= --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set serviceAccountName=kcontainer-runtime --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-ca-pemfile --set eventstreams.truststorePassword=password --output-dir templates --namespace eda-refarch chart/ordercommandms
```
Comment (jesusmah) [inline docs/deployments/application-components.md]: shouldnt `eventstreams.truststoreSecret=es-ca-pemfile` be `eventstreams.truststoreSecret=es-truststore-jks`

File: docs/deployments/application-components.md [hunk 2]
```diff
@@ -255,20 +256,25 @@ docker push <private-registry>/<image-namespace>/order-query-ms:latest
...
+    * `--set eventstreams.apikeyConfigMap=<kafka api key Secret name>`
+    * `--set eventstreams.truststoreRequired=(true/false)` (`true` when connecting to Event Streams via ICP4I)
+    * `--set eventstreams.truststoreSecret=<eventstreams jks file secret name>` (only used when connecting to Event Streams via ICP4I)
+    * `--set eventstreams.truststorePassword=<eventstreams jks password>` (only used when connecting to Event Streams via ICP4I)
+    * `--set serviceAccountName=<service-account-name>`
+    * `--namespace <target-namespace>`
+    * `--output-dir <local-template-directory>`
+  Example using Event Streams via ICP4I:
+   ```shell
+   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/order-query-ms --set image.pullSecret= --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set serviceAccountName=kcontainer-runtime --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-ca-pemfile --set eventstreams.truststorePassword=password --output-dir templates --namespace eda-refarch chart/orderqueryms
```
Comment (jesusmah) [inline docs/deployments/application-components.md]: shouldnt `eventstreams.truststoreSecret=es-ca-pemfile` be `eventstreams.truststoreSecret=es-truststore-jks`

File: docs/deployments/application-components.md [hunk 3]
```diff
@@ -314,20 +319,29 @@ docker push <private-registry>/<image-namespace>/kc-spring-container-ms:latest
...
+    * `--set postgresql.urlSecret=<postgresql url Secret name>`
+    * `--set postgresql.userSecret=<postgresql user Secret name>`
+    * `--set postgresql.passwordSecret=<postgresql password Secret name>`
+    * `--set serviceAccountName=<service-account-name>`
+    * `--namespace <target-namespace>`
+    * `--output-dir <local-template-directory>`
+  Example using Event Streams via ICP4I:
+   ```shell
+   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/kc-spring-container-ms --set image.pullSecret= --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-truststore-jks --set eventstreams.truststorePassword=password --set postgresql.capemRequired=true --set postgresql.capemSecret=postgresql-ca-pem --set postgresql.urlSecret=postgresql-url --set postgresql.userSecret=postgresql-user --set postgresql.passwordSecret=postgresql-pwd --set serviceAccountName=kcontainer-runtime --output-dir templates --namespace eda-refarch chart/springcontainerms
```
Comment (jesusmah) [inline docs/deployments/application-components.md]: I see
```
eventstreams.truststoreSecret
eventstreams.truststorePassword
```
but not
```
eventstreams.caPemFileRequired
eventstreams.caPemSecretName
```

File: docs/deployments/application-components.md [hunk 4]
```diff
@@ -447,13 +455,13 @@ docker push <private-registry>/<image-namespace>/kc-fleet-ms:latest
...
-  - Example using Event Streams via ICP4I:
+  Example using Event Streams via ICP4I:
    ```shell
-   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/kc-fleet-ms --set image.tag=latest --set image.pullSecret= --set image.pullPolicy=Always --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set serviceAccountName=kcontainer-runtime --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-ca-pemfile --set eventstreams.truststorePassword=password --output-dir templates --namespace eda-pipelines-sandbox chart/fleetms
+   helm template --set image.repository=rhos-quay.internal-network.local/browncompute/kc-fleet-ms --set image.pullSecret= --set kafka.brokersConfigMap=es-kafka-brokers --set eventstreams.enabled=true --set eventstreams.apikeyConfigMap=es-eventstreams-apikey --set serviceAccountName=kcontainer-runtime --set eventstreams.truststoreRequired=true --set eventstreams.truststoreSecret=es-ca-pemfile --set eventstreams.truststorePassword=password --output-dir templates --namespace eda-refarch chart/fleetms
```
Comment (jesusmah) [inline docs/deployments/application-components.md]: shouldnt `eventstreams.truststoreSecret=es-ca-pemfile` be `eventstreams.truststoreSecret=es-truststore-jks` ??

**Comment by jesusmah** [comment]
Also, I dont see instructions for creating the secrets `es-ca-pemfile` or ` es-truststore-jks`

**Comment by osowski** [comment]
I will fix and force merge the updates for the confusion between `eventstreams.truststoreSecret=es-ca-pemfile` and `eventstreams.truststoreSecret=es-truststore-jks` shortly.

**Pull request merged**